src/negotiation/order.ts

```diff
@@ -58,6 +79,14 @@ export function isNative({ asset, ledger }: OrderAsset): boolean {
 const BITCOIN_DECIMALS = 8;
 const ETHER_DECIMALS = 18;
 
+/**
+ * Convert from nominal amount to underlying base unit of an asset, e.g. convert Ether to Wei.
```

Comment:
Mention that it also works for Bitcoin

Reply:
It serves as an example here, I don't want to mention Bitcoin, then I also have to explain the token. I'd say we leave it like this for now, or just remove the example.